source/cloud/nvidia/brev.md

```diff
@@ -37,16 +37,25 @@ THere are two options to get you up and running with RAPIDS in a few steps, than
 
 ### Option 2. Setting up your Brev Launchable
 
-1. Go to **[Brev's Launchable Creator](https://brev.nvidia.com/launchables/create)** (requires account)
-2. Click **Compute** and select your GPU based on GPU type, number of GPUs, the cloud provider, and/or budget you have. It is good to understand your GPU requirements before picking an instance, or use it to figure out which instance. Once selected, click **Save**
-3. Click **Container**. When adding the **Container**, you can use the NVIDIA RAPIDS Container, use Docker Compose and edit our example yaml so that it preinstalls any additional conda or pip packages into your container before entry.
-4. If you just need standard NVIDIA RAPIDS install, Select **Container Mode > NVIDIA RAPIDS Container**. If using the Base Container, you may need to preinstall Jupyter. If using the Notebooks Container, **Do not** Preinstall Jupyter, as that will break your instance.
-5. If you need a customized environment, with additional packages on top of NVIDIA RAPIDSm use **Docker Compose** When using Docker Compose, you can upload a docker-compse yaml file. Here is an example docker-compose file, **[docker/brev/docker-compose-nb-2412.yaml](https://github.com/clara-parabricks-workflows/single-cell-analysis-blueprint/raw/main/docker/brev/docker-compose-nb-2412.yaml)** that you can use as your base. **Do not** Preinstall Jupyter when using that file, as it already will be installed.
-6. Click **Save**
-7. Click **Files** and add in any publicly available single file or github repository. Click **Save**
-8. In **Ports**, please open ports **8888, 8787, and 8786**. Name port 8888 `jupyter` so Brev can treat it as a Jupyter-Lab based instance and provide an **Open Notebook** button. Click **Save**
-9. Name your Launchable, then Save your Launchable!
-10. Whenever you're ready to use your Launchable, Select your Launchable and hit **Deploy Launchable**
+> [!WARNING]  
```

Comment:
please follow admonitions convention used in these docs see https://github.com/rapidsai/deployment/tree/main?tab=readme-ov-file#custom-admonitions